assets/common/js/models/Player.js

```diff
@@ -1,3 +1,24 @@
 function Player() {
-    
+    this.score = 0;
+    this.numCharsEntered = 0;
```

Comment:
You need to move this variable if you are moving the methods below.

Reply:
Please see my comment in the first conversation. I made a commit for stopping the game when player lives reaches zero. To test it, you can try manually setting the player lives variable value to zero as shown in the screenshot.

We have not implemented decrementing player's lives when they miss a word yet, so this is how I test it for now.


![Capture](https://user-images.githubusercontent.com/16367862/150052525-5ad7f782-dab6-4a24-b04a-ab6be531f05f.JPG)

Reply:
Okay :D